bench/wtperf/wtperf.c

```diff
@@ -603,7 +604,20 @@ worker(void *arg)
 op_err:			lprintf(cfg, ret, 0,
```

Comment:
I don't think a `WT_ROLLBACK` deserves a first-class error message from wtperf. If it is an expected case, it should be at least level 1.